src/Repository/DefaultUnleashProxyRepository.php

```diff
@@ -153,16 +153,17 @@ private function addQuery(string $url, string $query): string
      *             value: string
      *         }
      *     },
-     *     impression_data: bool
+     *     impression_data?: bool,
+     *     impressionData?: bool
```

Comment:
One will be set, the other won't, we don't know which one because it depends on the version of the backend